### PR TITLE
docs: Fix typo; editorial pass on Introduction

### PR DESCRIPTION
**Pull request by richb-hanover**
* Change heading to say "Querying with PRQL"
* Editorial tweaks to the Introduction

I don't know enough about polars or pandas to do more with the intro

### CODE REVIEW

File: docs/intro.qmd [hunk 1]
```diff
@@ -5,25 +5,24 @@ sidebar_position: 1
...
-At the same time, new features are being added to DuckDB's SQL on a constant basis,
-and the syntax is becoming more friendly.
+* [PRQL](https://prql-lang.org/) is a modern language for data manipulation
+that compiles to SQL.
+Although the PRQL language is still evolving,
+it is easy to create complex SQL queries. 
+* [DuckDB](https://duckdb.org/) is a modern in-memory
+([very fast](https://duckdblabs.github.io/db-benchmark/)) SQL OLAP
+database management system.
+DuckDB is also evolving, with new features and improving syntax.
```
Comment (eitsupi) [inline docs/intro.qmd:21]:
```suggestion
- [PRQL](https://prql-lang.org/) is a modern language for data manipulation that compiles to SQL.
  Although the PRQL language is still evolving, it is easy to create complex SQL queries. 
- [DuckDB](https://duckdb.org/) is a modern in-memory ([very fast](https://duckdblabs.github.io/db-benchmark/))
  SQL OLAP database management system.
  DuckDB is also evolving, with new features and improving syntax.
```

File: docs/intro.qmd [hunk 2]
```diff
@@ -5,25 +5,24 @@ sidebar_position: 1
...
 
-So, I thought that by comparing PRQL and the latest DuckDB SQL (and other query libraries),
-we could clarify the missing features of the PRQL and highlight the latest features of DuckDB.
+This project uses CLI, JavaScript, Python, R, etc.
```
Comment (eitsupi) [inline docs/intro.qmd:23]: I don't think I am using CLI or JS within this document.

I just wanted to say that the combination of PRQL and DuckDB can be used in many different places.
Could you change the wording to something else?

**Review by eitsupi — COMMENTED** [review]
Thanks for this PR!
A few comments.

File: docs/intro.qmd (hunk 2 above)
Comment (richb-hanover) [reply docs/intro.qmd]: Check the new wording of this paragraph

Comment (eitsupi) [reply docs/intro.qmd]: Sorry, but the new version seems to have completely removed the intent of the original document.
My point was that the combination of DuckDB and PRQL can be used in a variety of environments.

Comment (richb-hanover) [reply docs/intro.qmd]: Oh dear. I am sorry to have missed the meaning. I will close this PR, and I will watch for updates to the project.

**Pull request closed without merging**